lib/app/core/managers/app_configuration.dart

```diff
@@ -106,4 +114,27 @@ class AppConfiguration implements IAppConfiguration {
       hasActivedGuardian: data['hasActivedGuardian'],
     );
   }
+
+  Future<void> _clearUserData() async {
+    await Future.wait([
+      _clearCacheDir(),
+      _clearAppDir(),
+    ]);
+  }
+
+  Future<void> _clearCacheDir() async {
+    final cacheDir = await getTemporaryDirectory();
+
+    if (cacheDir.existsSync() && (Platform.isIOS || Platform.isAndroid)) {
+      cacheDir.deleteSync(recursive: true);
```

Comment:
Como os testes rodam no Linux, se essa linha for executada nos testes eles irão quebrar pois tenta remover o diretório `/tmp`, já em aplicativos o diretório temporário é referente a do próprio app.